GeometryTools/Rectangle.swift

```diff
@@ -192,6 +192,15 @@ public struct Rectangle: ConvexPolygonProtocol {
             )
         }
     }
+
+    /// - Returns: `Rectangle` geometrically equivalent to this one, with positive `height`
+    /// and `width`.
+    public func normalized() -> Rectangle {
```

Comment:
Because this is a function that takes no arguments, we should make it a computed property.

```Swift
var normalized: Rectangle {
    ...
}
```